test/async-hooks/test-graph.tls-write.js

```diff
@@ -20,8 +20,8 @@ hooks.enable();
 //
 const server = tls
   .createServer({
-    cert: fs.readFileSync(common.fixturesDir + '/test_cert.pem'),
-    key: fs.readFileSync(common.fixturesDir + '/test_key.pem')
+    cert: fs.readFileSync(`${common.fixturesDir}/test_cert.pem`),
```

Comment:
IMHO is these cases `path.join` is much better:
1. it's more correct &mdash; Windows uses `\` while other OSs use `/`, and `path.join` handles that
2. it's more expressive &mdash; you explicitly say "this is a path I'm working with here"
3. more readable &mdash; a little bit easier to see the parts that are being joined